packages/web-components/src/styles/partials/badge.partials.ts

```diff
@@ -0,0 +1,299 @@
+import { css } from '@microsoft/fast-element';
+import {
+  colorBrandBackground,
+  colorBrandBackground2,
+  colorBrandForeground1,
+  colorBrandForeground2,
+  colorBrandStroke2,
+  colorNeutralBackground1,
+  colorNeutralBackground4,
+  colorNeutralBackground5,
+  colorNeutralForeground1,
+  colorNeutralForeground3,
+  colorNeutralForegroundInverted,
+  colorNeutralForegroundOnBrand,
+  colorNeutralForegroundStaticInverted,
+  colorNeutralStroke2,
+  colorNeutralStrokeAccessible,
+  colorPaletteDarkOrangeBackground1,
+  colorPaletteDarkOrangeBackground3,
+  colorPaletteDarkOrangeBorder1,
+  colorPaletteDarkOrangeForeground1,
+  colorPaletteDarkOrangeForeground3,
+  colorPaletteGreenBackground1,
+  colorPaletteGreenBackground3,
+  colorPaletteGreenBorder2,
+  colorPaletteGreenForeground1,
+  colorPaletteGreenForeground2,
+  colorPaletteGreenForeground3,
+  colorPaletteRedBackground1,
+  colorPaletteRedBackground3,
+  colorPaletteRedBorder1,
+  colorPaletteRedForeground1,
+  colorPaletteRedForeground3,
+  colorPaletteYellowBackground1,
+  colorPaletteYellowBackground3,
+  colorPaletteYellowBorder1,
+  colorPaletteYellowForeground2,
+  colorTransparentStroke,
+  fontSizeBase100,
+  fontSizeBase200,
+  lineHeightBase100,
+  lineHeightBase200,
+  spacingHorizontalSNudge,
+  spacingHorizontalXS,
+  spacingHorizontalXXS,
+} from '../../theme/design-tokens.js';
+
+const textPadding = spacingHorizontalXXS;
+
+/**
+ * @public
+ * The badge's size styles
+ */
+export const badgeSizeStyles = css.partial`
+  :host([size='tiny']) {
+    width: 6px;
+    height: 6px;
+    font-size: 4px;
+    line-height: 4px;
+  }
+  :host([size='tiny']) ::slotted(svg) {
+    font-size: 6px;
+  }
+  :host([size='extra-small']) {
+    width: 10px;
+    height: 10px;
+    font-size: 6px;
+    line-height: 6px;
+  }
+  :host([size='extra-small']) ::slotted(svg) {
+    font-size: 10px;
+  }
+  :host([size='small']) {
+    min-width: 16px;
+    height: 16px;
+    font-size: ${fontSizeBase100};
+    line-height: ${lineHeightBase100};
+    padding-inline: calc(${spacingHorizontalXXS} + ${textPadding});
+  }
+  :host([size='small']) ::slotted(svg) {
+    font-size: 10px;
+  }
+  :host([size='medium']) {
+    min-width: 20px;
+    height: 20px;
+    font-size: ${fontSizeBase200};
+    line-height: ${lineHeightBase200};
+    padding-inline: calc(${spacingHorizontalXS} + ${textPadding});
+  }
+  :host([size='medium']) ::slotted(svg) {
+    font-size: 10px;
+  }
+  :host([size='large']) {
+    min-width: 24px;
+    height: 24px;
+    font-size: ${fontSizeBase200};
+    line-height: ${lineHeightBase200};
+    padding-inline: calc(${spacingHorizontalXS} + ${textPadding});
+  }
+  :host([size='large']) ::slotted(svg) {
+    font-size: 10px;
```

Comment:
```suggestion
    font-size: 16px;
```

Reply:
Good catch with turtles all the way down. :)